tests/functional/ext/docparams/docparams.py

```diff
@@ -102,3 +102,6 @@ def params_are_documented(par1: int, *, par2: int) -> int:
     """
 
     return par1 + par2
+
+
+raise Exception()
```

Comment:
This feels really weird without a comment. What does this test?

Reply:
https://github.com/PyCQA/pylint/blob/8912838a27f3391d78e11ed32ce40b58074d8d22/pylint/extensions/docparams.py#L278

Reply:
Unlike the other cases, this is not an impossible circumstance.